src/TriggerService/CromwellOnAzureEnvironment.cs

```diff
@@ -114,6 +115,44 @@ public async Task ExecuteNewWorkflowsAsync()
             }
         }
 
+        internal async Task<(string, byte[], List<string>, List<byte[]>, string, byte[], string, byte[])> ProcessBlobTrigger(string blobTriggerJson)
```

Comment:
Can you refactor this so it returns a class, such as BlobTriggerResult ?  Otherwise it's impossible to tell what this method returns without carefully reviewing the code, or relying on the variable names in the calling method.